From a75c3afe685e563f919a1a87e0a12c996cfae735 Mon Sep 17 00:00:00 2001
From: Will Woods <wwoods@redhat.com>
Date: Wed, 11 Jan 2012 12:48:28 -0500
Subject: [PATCH] minor cleanups in parsing for dmsquash-live and livenet

- use wait_for_dev in livenet
- clarify log message and fix missing newline in dmsquash-live

Signed-off-by: Will Woods <wwoods@redhat.com>
---
 modules.d/90dmsquash-live/parse-dmsquash-live.sh |    4 ++--
 modules.d/90livenet/parse-livenet.sh             |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/modules.d/90dmsquash-live/parse-dmsquash-live.sh b/modules.d/90dmsquash-live/parse-dmsquash-live.sh
index 920647e..5ccc0d0 100755
--- a/modules.d/90dmsquash-live/parse-dmsquash-live.sh
+++ b/modules.d/90dmsquash-live/parse-dmsquash-live.sh
@@ -43,9 +43,9 @@ case "$liveroot" in
     live:/*.[Ii][Mm][Gg]|/*.[Ii][Mm][Gg])
         [ -f "${root#live:}" ] && rootok=1 ;;
 esac
-info "liveroot was $liveroot, is now $root"
+info "root was $liveroot, is now $root"
 
 # make sure that init doesn't complain
 [ -z "$root" ] && root="live"
 
-wait_for_dev /dev/mapper/live-rw
\ No newline at end of file
+wait_for_dev /dev/mapper/live-rw
diff --git a/modules.d/90livenet/parse-livenet.sh b/modules.d/90livenet/parse-livenet.sh
index 323fd4a..2e6bc4b 100755
--- a/modules.d/90livenet/parse-livenet.sh
+++ b/modules.d/90livenet/parse-livenet.sh
@@ -16,4 +16,4 @@ case "$liveurl" in
         rootok=1 ;;
 esac
 
-echo '[ -e /dev/root ]' > $hookdir/initqueue/finished/livenet.sh
+wait_for_dev /dev/root
